packages/web/src/components/GraphQLHooksProvider.tsx

```diff
@@ -2,6 +2,9 @@ import type { DocumentNode } from 'graphql'
 
 export interface GraphQLHookOptions {
   variables?: Record<string, any>
+  refetchQueries?: { query: DocumentNode; variables?: Record<string, any> }[]
+  onCompleted?: () => any
```

Comment:
The suggested change got reverted @corbt after the force push :).

Reply:
Drat, that's what I get for trying to solve problems late at night 🤣 . Re-added.